apps/src/templates/progress/ProgressLesson.story.jsx

```diff
@@ -122,11 +123,48 @@ export default storybook => {
           <ProgressLesson
             {...defaultProps}
             lesson={fakeLesson('Asessment Number One', 1, true)}
-            levels={fakeLevels(5, {named: false}).map(level => ({
-              ...level,
-              status: LevelStatus.locked
-            }))}
-            lessonLockedForSection={() => true}
+            levels={fakeLevels(5, {named: false})}
+            lessonIsLockedForUser={(lesson, levels, viewAs) => {
+              return true;
+            }}
```

Comment:
```suggestion
            lessonIsLockedForUser={(lesson, levels, viewAs) => true}
```
or even just
```
() => true
```
here and below